Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'rails', '4.2.6'
 gem 'pg', '~> 0.15'
 gem 'sass-rails', '~> 5.0'
+gem 'rails-api'
```

Comment:
:+1: We should probably lock this down, especially since it's pre-1.0. `gem 'rails-api', '0.4.0' should do it.